### DISCUSSION TITLE
Migrating from Power Mail-in-a-Box (Dovecot): Importing old mails

### DISCUSSION THREAD

**Discussion by sgilissen** [postalserver/postal#2050] [Help with using Postal] [answered]
Hi there!

I'm contemplating migrating my Power Mail-in-a-Box mailserver (which is based on Dovecot) to Postal. 

I've installed Postal on a new server, and plan to transition over once everything is set up and running. 

Is it possible to import old mails from Dovecot into Postal? 
I did manage to export backups (in Text (message/rfc822) format) from the old mail server.
What would be the easiest way to tackle this?

Thanks in advance!

**Comment by uhlhosting** [comment, marked as answer]
Hi, 

I just want to say that Postal is not really a mailbox, in a sense is more like a Sendgrid, Mailgun service, that you can use on as many domains as you please to generate credentials and send emails and see their stats. Hope this helps.

**Comment by sgilissen** [comment]
Ah, that makes sense. I'll stick to PMIAB then. Thanks!
